views/ancestorLines/ancestor_lines_explorer.js

```diff
@@ -0,0 +1,499 @@
+            </li>
+            <li>
+            The <b>Width</b> specifies the width of the display area. As the tree is drawn to the right, anything that
+            does not fit into this width, will be clipped.
+            </li>
+            <li>
+            The <b>Show tree eto level</b> value determines how many levels of the tree will be filled in with people that
+            are not in a line to a person of interest. If you set this value to 0, the complete tree will be shown
+            (subject to the setting of the other parameters).
+            </li>
+        </ul>
+        <p>
+            When you draw a complete tree (or any other time too!) experiment with the above values untill you find optimal
+            values for readablitiy of your tree given its size. You can specify big numbers. For example a height of 15000
+            and height of 3000 worked ok for me on a request of 12 generations that resulted in a tree with 16 levels. Just
+            be prepared to scroll (or drag) the screen to see everything.
+        </p>
+        <p>
+            If you find problems with this app or have suggestions for improvements, please
+            <a style="color: navy; text-decoration: none" href="https://www.wikitree.com/wiki/Smit-641" , target="_blank"
+            >let me know</a>.
+        </p>
+        <p>You can double click in this box, or click the X in the top right corner to remove this About text.</p>
+   `;
+
+    constructor(selector, startId) {
+        this.selector = selector;
+        $(selector).html(`<div id="aleContainer" class="ale">
+            <div id="controlBlock">
+            <label for="generation">Max Generations:</label
+            ><select id="generation" title="The number of generations to retrieve">
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+                <option value="5">5</option>
+                <option value="6">6</option>
+                <option value="7">7</option>
+                <option value="8">8</option>
+                <option value="9">9</option>
+                <option value="10">10</option>
+                <option value="11" selected>11</option>
+                <option value="12">12</option>
+                <option value="13">13</option>
+                <option value="14">14</option>
+                <option value="15">15</option>
+                <option value="16">16</option>
+                <option value="17">17</option>
+                <option value="18">18</option>
+                <option value="19">19</option>
+                <option value="20">20</option>
+            </select>
+            <label for="wtid">Start Profile:</label
+            ><input id="wtid" type="text" placeholder="Enter WikiTree ID" value="${
```

Comment:
why are some views (including this one) using duplicit fielnd for profile ID and not the global one?
![image](https://github.com/wikitree/wikitree-dynamic-tree/assets/20949562/46a28593-b07d-40de-badb-d5acf01746dc)